.github/workflows/publish-nuget.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish NuGet Packages
+
+on:
+  push:
+    tags:
+      - '[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Ah wait, we _cant_ publish a preview package - we're filtering on only "normal" tags in here! :D

Can we trigger on something else? Either the full semver-tags (`\d+.\d+.\d+(-.+)` IIRC) or just on any release?

Reply:
I updated the filter to also react to these pre-release tags with a (any) suffix.